src/lib/dataElements/ecid/index.js

```diff
@@ -10,22 +10,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-module.exports = (window, runAlloy, imsOrgId) => {
-  const { instances } = turbine.getExtensionSettings();
-  const names = instances.map(instance => instance.name);
+const createEcid = require("./createEcid");
+const instanceManager = require("../../instanceManager/index");
 
-  runAlloy(names);
-
-  instances.forEach(({ name, ...options }) => {
-    window[name]("configure", {
-      ...options,
-      imsOrgId
-    });
-  });
-
-  return {
-    getInstance(name) {
-      return name ? window[name] : undefined;
-    }
-  };
-};
+module.exports = createEcid(instanceManager);
```

Comment:
Git shows this diff as `createInstanceManager.js` being moved/renamed to `ecid/index.js`, which isn't the case. `ecid/index.js` is a new file.